bindings/nodejs/lib/wallet/account.ts

```diff
@@ -1190,8 +1203,8 @@ export class Account {
      * @returns The sent transaction.
      */
     async send(
-        amount: bigint | string,
-        address: string,
+        amount: bigint | NumericString,
+        address: HexEncodedString,
```

Comment:
```suggestion
        address: Bech32Address,
```
Probably also needs an import